packages/amplify-graphql-function-transformer/src/graphql-function-transformer.ts

```diff
@@ -178,6 +193,51 @@ export class FunctionTransformer extends TransformerPluginBase {
   };
 }
 
+/**
+ * The response mapping template for 'RequestResponse' (default) and 'Event' invocation types differ.
+ * Use this to generate the appropriate response mapping template.
+ * @param config the {@link FunctionDirectiveConfiguration} for `@function` definition.
+ * @returns the response mapping template used by AppSync to handle responses from the Lambda function invocation.
+ */
+const responseMappingTemplate = (config: FunctionDirectiveConfiguration): string => {
+  if (config.invocationType === 'Event') {
+    /*
+      #set( $success = true )
+      #if( $ctx.error )
+        $util.appendError($ctx.error.message, $ctx.error.type)
+        #set( $success = false )
+      #end
+      #set( $response = {
+        "success": $success
+      } )
+      $util.toJson($response)
+    */
+    return printBlock('Handle error or return result')(
```

Comment:
Should we add detail like "for an asynchronous function invocation"? Might make it easier to read through the generated VTL.

Reply:
Same as above. Addressed in https://github.com/aws-amplify/amplify-category-api/pull/2868